scipy/optimize/tests/test_slsqp.py

```diff
@@ -305,6 +306,11 @@ def test_integer_bounds(self):
         # This should not raise an exception
         fmin_slsqp(lambda z: z**2 - 1, [0], bounds=[[0, 1]], iprint=0)
 
+    def test_obj_must_return_scalar(self):
+        # If objective function does not return a scalar, raises ValueError
```

Comment:
Nitpick: add something like "Regression test for gh-5433" so that someone reading the code can easily find the context.